src/index.js

```diff
@@ -265,7 +269,7 @@ const generateCriticalCssWrapped = async function generateCriticalCssWrapped (
         } else {
           console.log('restarting chrome after crash')
           browser = null
-          await launchBrowserIfNeeded(debuglog)
+          await launchBrowserIfNeeded(options, debuglog)
```

Comment:
it's not safe to try to re-use the browser you launched in this case, as it (chrome) has crashed.

Reply:
The easiest way forward with the logic you set up would be to pass `{}` instead of options here. I might clean up the internal logic later, but should be good enough for now.

Reply:
However that would mean that the browser that restarts does not have the setup you require.